A/ABC/build_tarballs.jl

```diff
@@ -0,0 +1,79 @@
+using BinaryBuilder, Pkg
+
+# abc
+name = "ABC"
+version = v"1.01.0"
+ABC_ver = "1.01.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/berkeley-abc/abc.git", "645752f7d651b07a0ebd7cac827d6ce5dd5bd266")
+]
+
+# ABC is setup for native builds (the arch_flags program and running the
+# shell to get other flags), so we need to manually add the arch flags
+# here.  Also, the combo of BinaryBuilder and ABC's Makefile always seems
+# to override rather than extend CFLAGS, so I've included the whole mess
+# here.  Even though we use ABC_USE_STDINT_H=1 it doesn't work as
+# expected, so we only use it here to prevent the Makefile from running
+# the arch_flags executable, which will always fail when cross compiling.
+# Note that neither -DLIN nor -DLIN64 actually appear to be used anymore but
+# still set here anyway
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+EXFLGS="-Wall -Wno-unused-function -Wno-write-strings -Wno-sign-compare -DSIZEOF_INT=4 -DABC_USE_CUDD=1 -DABC_USE_PTHREADS -Wno-unused-but-set-variable -fPIC"
+
+if [[ ${nbits} == 32 ]]; then
+    EXFLGS="${EXFLGS} -DSIZEOF_VOID_P=4"
+    EXFLGS="${EXFLGS} -DSIZEOF_LONG=4"
+    EXFLGS="${EXFLGS} -DLIN"
+else
+    EXFLGS="${EXFLGS} -DSIZEOF_VOID_P=8"
+    EXFLGS="${EXFLGS} -DSIZEOF_LONG=8"
+    EXFLGS="${EXFLGS} -DLIN64"
+fi
+
+# musl doesn't like readline
+if [[ ${target} != *"musl"* ]]; then
+    EXFLGS="${EXFLGS} -DABC_READLINE"
+fi
+
+cd ${WORKSPACE}/srcdir/abc
+
+# it appears necessary to have the file(s) for the FileProduct(s) live in the destdir
+cp abc.rc ${WORKSPACE}/destdir
```

Comment:
For reference: https://docs.binarybuilder.org/stable/build_tips/#Build-strategy.  So yes, all files that will go into the tarball must be within the prefix, but you can move it wherever you want, the first argument to `FileProduct` will be its path within the prefix.
```suggestion
# it appears necessary to have the file(s) for the FileProduct(s) live in the prefix
cp abc.rc ${prefix}/.
```

Reply:
Done (and thanks).  I actually read that section multiple times and it just didn't click.  Reading it now and it seems obvious.  That's always the way, isn't it?  BTW, when there is a missing build product, BB exits rather than dropping into the debug shell.  Is that intentional?